README.md

```diff
@@ -1,2 +1,4 @@
 # mygithubdemo
 Help my friends understand git and github
+
+Add something here
```

Comment:
You should be more descriptive with your commit message will! Might I recommend "Add a more descriptive message here" instead of "Add something here".